ros_ign_point_cloud/examples/depth_camera.sdf

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" ?>
 <!--
-  Example publishing point cloud messages to ROS from Ignition.
+  Example publishing point cloud messages to ROS from Gazebo.
 
   Try it out:
 
   1. Start ROS core:
 
           roscore
 
-  2. Make sure Ignition Gazebo can find the point cloud plugin by adding your lib install directory
+  2. Make sure Gazebo Sim can find the point cloud plugin by adding your lib install directory
      to the path, for example:
 
+          export GZ_SIM_SYSTEM_PLUGIN_PATH=<ws>/install/lib/
           export IGN_GAZEBO_SYSTEM_PLUGIN_PATH=<ws>/install/lib/
```

Comment:
Just the new one should be enough, we shouldn't encourage users to use the old one anymore. The same goes for other files.

Reply:
Is the the case even if those env vars aren't supported by distributions pre-Garden?

Reply:
I'll remove IGN_GAZEBO_SYSTEM_PLUGIN_PATH anyway

Reply:
Ahh good call, I forgot that this branch supports Fortress. We'll backport the env vars upstream soon 🤞🏽